app/stores/ada/AdaWalletsStore.js

```diff
@@ -127,6 +131,9 @@ export default class AdaWalletsStore extends WalletStore {
     this.restoreRequest.reset();
     await this._patchWalletRequestWithNewWallet(restoredWallet);
     this.refreshWalletsData();
+
+    // show success notification
+    this.showWalletRestoredNotification();
```

Comment:
there is an extra space for some weird reason 🤔

Reply:
If you are talking about line number 134 then it's intentional.

I always try to group related things together and separate each group by one line.
I feel that way is more readable.

below is an example of how Trezor separate:
![image](https://user-images.githubusercontent.com/19986226/50276375-26d27900-0485-11e9-9e6c-97df646a2307.png)
I feel it's quite readable.

Reply:
it was this 
<img width="713" alt="screen shot 2018-12-20 at 3 10 34 pm" src="https://user-images.githubusercontent.com/1622112/50308504-7014cf80-0469-11e9-8003-179bbfef6e5f.png">

the spacing in 130-131, but I don’t see it in the actual code 🤔

Reply:
I guess it’s a github formatting thing. I haven never noticed it before